src/common/dao/project/projectmember.go

```diff
@@ -70,6 +70,59 @@ func GetProjectMember(queryMember models.Member) ([]*models.Member, error) {
 	return members, err
 }
 
+// GetTotalOfProjectMembers returns total of project members
+func GetTotalOfProjectMembers(projectID int64, roles ...int) (int64, error) {
+	log.Debugf("Query condition %+v", projectID)
+	if projectID == 0 {
+		return 0, fmt.Errorf("failed to get total of project members, project id required %v", projectID)
+	}
+
+	sql := `
+SELECT
+  COUNT(1)
+FROM
+  (
+    SELECT
+      pm.id AS id,
+      pm.project_id AS project_id,
+      r.role_id AS role
+    FROM
+      user_group ug
```

Comment:
The group can be a LDAP group, and you cannot list users under the group. So should we show the users under the group or just show the count of users + groups?

Reply:
Fixed